truckersmp-cli

```diff
@@ -470,6 +471,25 @@ Please report an issue: {}""".format(e, URL.issueurl))
             sys.exit("Failed to download mod files.")
 
 
+def get_supported_game_versions():
+    """Get TruckersMP-supported game versions via TruckersMP Web API."""
+    # Returns a dict of 'game: version' pairs:
+    # {
+    #     "ets2": "1.36.2.55",
+    #     "ats": "1.36.1.40"
+    # }
+    try:
+        with urllib.request.urlopen(URL.truckersmp_api) as f:
+            data = json.load(f)
+
+        return {
+            "ets2": data["supported_game_version"].replace("s", ""),
+            "ats": data["supported_ats_game_version"].replace("s", "")
+        }
+    except Exception:
+        pass
```

Comment:
If something goes wrong when getting information via the Web API, this function returns `None` because of "reaching end of function without `return` statement". I think this behavior itself is appropriate but I'd like `return None` instead of this `pass`.